aspnet/migration/rc1-to-rc2.rst

```diff
@@ -180,36 +97,31 @@ WebApplicationConfiguration    WebHostConfiguration
 
 The ``commands`` section of ``project.json`` has been removed completely. Use ``dotnet run`` or ``dotnet <dllname>`` instead.
 
+Hosting Configuration 
+---------------------
+You enable configuration for your web application host by calling the ``UseConfiguration()`` method on ``WebHostBuilder`` and passing a configuration instance. The following configuration values can be specified:
 
-IHostingEnvironment changes 
----------------------------
+-application
```

Comment:
This isn't rendering as a [list](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#bullet-lists) correctly. You have to have a space after the `-`.